src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -0,0 +1,74 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.codeu.data.Datastore;
+import com.google.codeu.data.User;
+
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+/**
+* Third Test - Fetches and saves user data
+*/
+@WebServlet("/about")
+public class AboutMeServlet extends HttpServlet{
+  private Datastore datastore;
+
+  @Override
+  public void init(){
+    datastore = new Datastore();
+  }
+
+  /**
+  * Responds with "about me" section for the user
+  */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+    throws IOException{
+
+      response.setContentType("text/html");
+
+      String user = request.getParameter("user");
+      if(user == null || user.equals("")){
+        response.getOutputStream().println("INVALID RESPONSE: STOP HACKING THE MAINFRAME");
```

Comment:
It would probably be better to redirect to 'index.html' or perhaps fallback to the current user from the UserService here and in a few error cases below. My main thinking here is that we don't really know how or why the user is in this situation. From their perspective they just want to be taken back to safety. From an admin perspective, it may be useful to log these errors to a log system or System.err so we could tell if a lot of users are making this mistake.

Reply:
Implemented in commit 6c5c5b8c56dd2a03548a89c3b9e743e249584730